SSPS/src/SSPS.jl

```diff
@@ -31,8 +59,8 @@ function parse_script_arguments()
         "output_path"
             help = "name of output JSON file"
             required = true
-        "time-per-chain"
-            help = "execution timeout (in seconds). When reached, terminate and output results as they are."
+        "timeout"
+            help = "execution timeout (in seconds). When reached, terminate and output results as they are. SSPS attempts to divide "
```

Comment:
Was the "SSPS attempts to divide " sentence cut off?

Reply:
My bad, I need to remove that sentence entirely